numba/parfor.py

```diff
@@ -2469,32 +2559,56 @@ def remove_dead_parfor(parfor, lives, arg_aliases, alias_map, typemap):
     """ remove dead code inside parfor including get/sets
     """
 
+    # add dummy return to last block for CFG computation
+    # max is last block since we add it manually for parfors
+    last_label = max(parfor.loop_body.keys())
+    parfor.loop_body[last_label].body.append(
```

Comment:
Same conversion to "with" style I suggest above.

Reply:
done.